File: Web_API/src/test/java/test/BaseTest.java
package test;

import aquality.selenium.browser.AqualityServices;
import aquality.selenium.browser.Browser;
import aquality.selenium.elements.interfaces.IElementFactory;
import kong.unirest.Unirest;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.BeforeMethod;
import utils.UtilsGetConfig;

public abstract class BaseTest {
    protected final IElementFactory elementFactory;

    protected BaseTest() {
        elementFactory = AqualityServices.getElementFactory();
    }

    @BeforeMethod
    protected void beforeMethod() {
        getBrowser().goTo(UtilsGetConfig.getConfig("defaultURL"));
        getBrowser().maximize();
        getBrowser().waitForPageToLoad();
        Unirest.config().connectTimeout(100000);
        Unirest.config().defaultBaseUrl(UtilsGetConfig.getConfig("defaultURLResponse"));
    }

    @AfterMethod
    public void afterTest() {
        if (AqualityServices.isBrowserStarted()) {
            AqualityServices.getBrowser().quit();
        }
    }

    protected void navigate(String url) {
        getBrowser().goTo(url);
    }

    protected Browser getBrowser() {
        return AqualityServices.getBrowser();
    }
}

File: Web/UserInterfaceTest/UserInterfaceTest/Templates/BaseTest.cs
using NUnit.Framework;
using Aquality.Selenium.Browsers;
using UserInterfaceTest.Configuration;

namespace UserInterfaceTest.Templates
{
    public abstract class BaseTest
    {
        protected Browser browser = AqualityServices.Browser;

        [SetUp]
        public void SetUp() 
        {
            
            browser.Maximize();
            browser.GoTo(Configuration.Configuration.StartUrl);
            browser.WaitForPageToLoad();
        }

        [TearDown]
        public void TearDown() 
        {
            browser.Quit();
        }

    }
}
